diff --git a/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue b/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue
index 8f2e95c9f..6359daad8 100644
--- a/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue
+++ b/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue
@@ -438,7 +438,7 @@
             },
             getStoreFrontUrl: function(pageName) {
                 //TODO: baseUrl should be maintain in configuration file
-                var baseUrl = "https://demo-in.hotwax.co/";
+                var baseUrl = "https://demo.hotwax.co/";
                 //TODO: We should work on better approach here to find url for the index page
                 if (pageName === "index") {
                     return baseUrl;
diff --git a/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/subnav/template.vue b/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/subnav/template.vue
index 13714d1ac..5639e795d 100644
--- a/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/subnav/template.vue
+++ b/admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/subnav/template.vue
@@ -60,7 +60,7 @@ export default {
                 if (this.$root.$data.pageView) {
                     if(this.$root.$data.pageView.path) {
                         //TODO: baseUrl should be maintain in configuration file
-                        var baseUrl = "https://demo-in.hotwax.co";
+                        var baseUrl = "https://demo.hotwax.co";
                         var path = this.$root.$data.pageView.path;
                         var pageName= path.substring(path.lastIndexOf("/"),path.length);
                         //TODO: We should work on better approach here to find url for the index page
